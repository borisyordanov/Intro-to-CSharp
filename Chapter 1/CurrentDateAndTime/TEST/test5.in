1998
5
24
11
39
5
