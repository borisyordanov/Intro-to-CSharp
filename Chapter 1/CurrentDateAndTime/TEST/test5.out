5/24/1998 11:39:05 AM
